src/handler-context/types.ts

```diff
@@ -2,11 +2,12 @@ import { Logger } from "@aws-lambda-powertools/logger";
 import { ConfigElements, PickedConfigCache } from "./config";
 import { HandlerOutputs } from "./context-builder";
 
-export type HandlerMessageBody = string | {};
+export type HandlerMessageBody = unknown;
```

Comment:
Because of the return type of these four functions: `makeS3Messages`, `makeSQSMessages`, `SQSMessageIncludesS3EventRecord` and `makeIncomingMessages` in the make-incoming-messages.ts file. This was how I was able to make it work. Happy for suggestions!

Reply:
Mark S gave me a crash course in generics on a call just now so I think I'm able to change this back to string | {}.